packages/react-dom/src/__tests__/ReactDOMServerIntegrationUntrustedURL-test.js

```diff
@@ -195,7 +204,6 @@ describe('ReactDOMServerIntegration - Untrusted URLs - disableJavaScriptURLs', (
     ReactFeatureFlags.disableJavaScriptURLs = true;
 
     React = require('react');
-    ReactDOM = require('react-dom');
```

Comment:
You'll need to keep this, because it's returned from initModules below, which is passed to `ReactDOMServerIntegrationUtils`